packages/chips/README.md

```diff
@@ -13,13 +13,13 @@ npm install @material/react-chips
 ### Styles
 
 with Sass:
-```js
-import '@material/react-chips/index.scss';
+```css
```

Comment:
Oh. I see. It wasn't obvious from the documentation, as it would require css-loader or sass-loader. But it would allow for easy de-duplication across modules, so it makes a lot of sense to place it in the JS. 👍